### DISCUSSION TITLE
Support to use it with sqlite?

### DISCUSSION THREAD

**Discussion by richerfu** [meilisearch/product#647] [Feedback & Feature Proposal]
I  want to use `sqlite` save some datas and use it to search.It's possible to use it with sqlite file?

**Comment by gmourier** [comment]
Hey @southorange0929 👋

No, it's not possible. Is there a particular reason not to send those data to Meilisearch and keep them to sqlite for search?

SQL databases may have full-text search features, but they are far from equaling Meilisearch, which is dedicated to search.

**Reply by richerfu** [reply]
Yes,you are right!Thanks for your reply
